src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationServiceTest.java

```diff
@@ -367,6 +370,29 @@ void shouldValidateJurisdictionCodesWithUniqueCodes() {
         assertEquals(0, errors.size());
     }
 
+    @Test
+    void validateCurrentPositionCaseClosed() {
+        CaseDetails caseDetails = new CaseDetails();
+        caseDetails.setState(CLOSED_STATE);
+        CaseData caseData = new CaseData();
+        caseData.setCurrentPosition(CASE_CLOSED_POSITION);
+        caseDetails.setCaseData(caseData);
+        boolean validated = eventValidationService.validateCurrentPosition(caseDetails);
+        assertTrue(validated);
+    }
+
+    @ParameterizedTest
```

Comment:
This is a good use of ParameterizedTest. Could you enhance what you've done so that you only need one unit test for your validateCurrentPosition method? You'd need two parameters for the single unit test - 1) the state 2) expected value. You'd then need to change the parameterized test to provide 2 values rather than 1.